src/main/java/step4/model/generator/LottoAutoGenerator.java

```diff
@@ -0,0 +1,48 @@
+package step4.model.generator;
+
+import step4.constant.LottoConstant;
+import step4.model.Lotto;
+import step4.model.LottoBuyCount;
+import step4.model.LottoNumber;
+import step4.model.Lottos;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class LottoAutoGenerator implements LottoGenerator {
+
+    private static final List<LottoNumber> lottoCandidateNumbers;
+
+    static {
+        lottoCandidateNumbers = LottoNumber.getLottoCandidateNumbers();
+    }
```

Comment:
위에서 코멘트 해드려서 아시겠지만 
꼭 필요한 경우가 아니라면 직접 할당해도 좋을 것 같습니다!! 😆
```suggestion
    private static final List<LottoNumber> lottoCandidateNumbers = LottoNumber.getLottoCandidateNumbers();
```